src/app/components/AllOrNone.tsx

```diff
@@ -24,6 +24,8 @@ export default styled(AllOrNone)`
   &,
   ${ButtonLink} {
     font-size: 1.4rem;
+    margin: 0;
```

Comment:
In the `ChapterFilter` line 115 there is also margin for `AllOrNone` which happens to have bigger priority so this `margin: 0` is making a difference.
I think you can remove this line, it looks good without this.